src/module/customer/domain/customer.entity.ts

```diff
@@ -0,0 +1,18 @@
+export enum DocumentTypeEnum {
+	dni = "dni",
+	passport = "passport",
+}
+
+export default class Customer {
+	id?: number;
+	name?: string;
+	lastName?: string;
+	documentType?: DocumentTypeEnum;
+	documentNumber?: number;
+	birthrate?: Date;
```

Comment:
perhaps this was meant to be `birthDate`?

Reply:
Sure. That's more accurate